webapp/tsconfig.json

```diff
@@ -16,6 +16,7 @@
         "emitDecoratorMetadata": true,
         "declaration": false,
         "preserveConstEnums": true,
+        "downlevelIteration": true,
```

Comment:
@pelikhan use of spread operator on map values required the addition of this compiler option (due to targeting es5). Any issue?

Reply:
You'd have to ask this to the language gurus, @riknoll @mmoskal . we should decide where to draw the line w.r.t. to typescript features, specially because we might start to confuse STS vs TS versions.
imo: not worth it.

Reply:
I'll revert the change to avoid risk. Curious what the language gurus think though.

Reply:
I think it should be fine. There's lots of things we do in TS that we cannot do in STS (like async).